### PR TITLE
Fix TensorFlow_Found variable in CMake

### PR DESCRIPTION
**Pull request by Semyon1104**

### CODE REVIEW

File: include/alexnet/CMakeLists.txt [hunk 1]
```diff
@@ -30,18 +30,19 @@ if(UNIX)
...
     find_package_handle_standard_args(tensorflow DEFAULT_MSG TensorFlow_INCLUDE_DIRS TensorFlow_LIBRARIES)
-endif()
 
-if(TENSORFLOW_FOUND)
-    message(STATUS "TensorFlow_Found")
-  else()
-    message(STATUS "TensorFlow library not found")
-  endif()
+    if(EXISTS "${TensorFlow_INCLUDE_DIRS}" AND EXISTS "${TensorFlow_LIBRARIES}")
+        message(STATUS "TensorFlow ihyhgfhvhjvhvhvhjvfound")
```
Comment (allnes) [inline include/alexnet/CMakeLists.txt:41]: please change messages

**Review by allnes — APPROVED** [review]

File: include/alexnet/CMakeLists.txt [hunk 2]
```diff
@@ -30,18 +30,19 @@ if(UNIX)
...
 
-if(TENSORFLOW_FOUND)
-    message(STATUS "TensorFlow_Found")
-  else()
-    message(STATUS "TensorFlow library not found")
-  endif()
+    if(EXISTS "${TensorFlow_INCLUDE_DIRS}" AND EXISTS "${TensorFlow_LIBRARIES}")
+        message(STATUS "TensorFlow found")
+    else()
+        message(STATUS "TensorFlow not found")
```
Comment (aobolensk) [inline include/alexnet/CMakeLists.txt:43]:
```suggestion
        message(WARNING "TensorFlow not found")
```

**Review by aobolensk — CHANGES_REQUESTED** [review]

**Comment by aobolensk** [comment]
Please, update PR name and maybe add description

**Comment by aobolensk** [comment]
> Please, update PR name and maybe add description

Updated

**Review by aobolensk — APPROVED** [review]

**Comment by aobolensk** [comment]
@NeiroYT please, review

**Review by NeiroYT — APPROVED** [review]

**Pull request merged**